sandbox.html

```diff
@@ -32,21 +34,28 @@ <h1 v-if="message.length % 2 == 0">{{ message }}</h1>
         const app = createApp({
             data() {
                 return {
-                    message: 'Hello World!',
-                    show: true,
-                    list: [1, 2, 3]
+                    message: 'Favourite Movie Series: Harry Potter',
+                    list: ['Harry', 'Ron','Hermione'],
+                    favoriteList: [],
+                    newItem: ''
                 }
             },
             methods: {
-                evaluate() {
-                    console.log('Evaluate')
+                addItem() {
+                    if (this.newItem.trim() !== '') {
```

Comment:
Normally `this` refers to the current object - so it looks for the `newItem` - It is the only way to to track the variables inside data in methods. But this will be changed when we look at composition API :D